### DISCUSSION TITLE
PC version is no longer working, would you have another option?

### DISCUSSION THREAD

**Discussion by batatajedai** [thatskymod/Sky-CotL-Scripts#95] [General]
I was just using the Daily Routine to farm faster. But today when I went to run the program it is no longer working, in the past it stuck at 98% but I received the candles. Now the program only closes by itself before it even reaches 98%, would there be any solution or other program just to farm daily?

**Comment by thatskymod** [comment]
Mods in this repository are regularly tested for compatibility. That said, I don't see any apparent issue with the current PC version. Be sure to update ThatSkyMod to the latest version, [3.5.0](https://github.com/XeTrinityz/ThatSkyMod/releases/tag/3.5.0), released two days ago. This update introduces several fixes and improvements over previous versions.
